conf/db/upgrade/V1.9__schema.sql

```diff
@@ -284,4 +284,6 @@ ALTER TABLE ShareableVolumeVmInstanceRefVO ADD CONSTRAINT fkShareableVolumeVmIns
 
 ALTER TABLE `LocalStorageResourceRefVO` DROP INDEX `resourceUuid`;
 ALTER TABLE `LocalStorageResourceRefVO` DROP PRIMARY KEY;
+ALTER TABLE `LocalStorageResourceRefVO` DROP FOREIGN KEY `fkLocalStorageResourceRefVOHostEO`;
```

Comment:
it seems to be a compatibility problem between mariadb 5.5 and 10.0 (ubuntu 16.04), the latter has stricter check